app/commands/decidim/anonymous_codes/admin/create_code_group.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Decidim
+  module AnonymousCodes
+    module Admin
+      class CreateCodeGroup < Decidim::Command
+        def initialize(form)
+          @form = form
+        end
+
+        def call
+          return broadcast(:invalid) if @form.invalid?
+
+          transaction do
+            create_code_group!
+          end
+
+          broadcast(:ok)
+        end
+
+        private
+
+        attr_reader :code_group, :form
+
+        def create_code_group!
+          @code_group = Decidim.traceability.create!(
+            Group,
+            @form.current_user,
```

Comment:
```suggestion
            form.current_user,
```
Better be consistent, as you have `attr_reader` defining `:form` you can use it safely everywhere instead of `@form`. Same into the `call` method

Reply:
Thanks, done.